### ISSUE TITLE
Tensorboard does not show accuracy

### ISSUE THREAD

**Issue by NinjaWendy** [Qberto/ML_ObjectDetection_CAFO#6]
Thank you for these guiding steps. I have finally made the model learning and have generated the inference graph. When I open Tensorboard I can only see the learning rate and loss. I would like to know the accuracy. I have delved into the script and found that the trainer.py (called from train.py) contains functions such as loss and sums it up for Tensorboard. Do you know how I can include the accuracy here? 
The loss is defined specifically for the model and comes back multiple times in the trainer.py script. Copy/paste accuracy from other script does not work so far.
